### ISSUE TITLE
TREE CONTROL - Keyboard user is unable to access the “...“ button in "Tree" Control using tab navigation.

### ISSUE THREAD

**Issue by sagaranandani17** [OfficeDev/office-ui-fabric-react#11061]
### Environment Information

- **Package version(s)**: 7.24.0
- **OS version:** 1903 (OS Build: 18326.295)
- **Browser:** Edge(Microsoft Edge 44.18362.267.0)

**User Experience:** 
If Tree controls (...) is not accessible with keyboard than keyboard user will not be able to access the control and will face difficulties while interacting with it.

**Actual behavior:**
Keyboard user is unable to access the “More options (...)“ button in tree control

**Expected behavior:**
Keyboard user should be able to access the “More options (...)“ button in tree control, using tab navigation.

**Please find the attachment**
![image](https://user-images.githubusercontent.com/57341535/68119684-56696300-ff29-11e9-9f9b-112029ee7cc6.png)

**Comment by JasonGore** [comment]
There is no such Fabric control as Tree. We need more information and a CodePen to properly triage this against a specific Fabric control.

**Comment by sagaranandani17** [comment]
We are using Tree control from Business app fabric and it's url is - https://businessappfabric.azurewebsites.net/#component=id__16

**Comment by JasonGore** [comment]
You have linked a different repo. The [official Fabric repo](https://github.com/OfficeDev/office-ui-fabric-react) and [Controls docs](https://developer.microsoft.com/en-us/fabric#/controls/web) do not contain a Tree. We can help troubleshoot issues, but we need a CodePen repro using official Fabric code to know which Fabric controls are being used.
